### DISCUSSION TITLE
Are there any plans to support JDK 11 and up

### DISCUSSION THREAD

**Discussion by vkras** [ikvmnet/ikvm#288] [Ideas]
I was trying to port Carrot2 library but it's compiled with Java 11:
Unable to compile class "org.carrot2.attrs.AcceptingVisitor" (55.0)

**Comment by wasabii** [comment]
Plan, as in wishes, sure. There is no concrete work being done on it directly however. It's a long road.

**Reply by wasabii** [reply]
My goal has initially to be to clean up the existing code base. Minimize deviations from OpenJDK code. To eventually make any such work easier.

**Comment by wasabii** [comment]
#120

**Reply by vkras** [reply]
Thank you for the response.

**Comment by GeorgeS2019** [comment]
I am still waiting for  [jdk8u111](https://github.com/ikvm-revived/ikvm/pull/265#issuecomment-1426820372)
